src/components/Orders.js

```diff
@@ -15,16 +15,6 @@ class Orders extends React.Component {
   }
 
   render() {
-    const headings = [
```

Comment:
did you mean to make this change? is it no longer needed?

Reply:
we don't use those variable for react table. I thought I removed that before.